templates/managed-batch.yaml

```diff
@@ -168,6 +173,7 @@ Resources:
         MinvCpus: 0
         DesiredvCpus: 0
         MaxvCpus: 64
+        ImageId: !If [HasAMIId, !Ref AMIId, !Ref "AWS::NoValue"]
```

Comment:
Do you know if the cloud formation template will work if there is no value?

Reply:
i have not tested but we do have a test that will tell us. https://github.com/Sage-Bionetworks/aws-infra/blob/master/ci/managed-batch-input.json

the test does not set this param so if the test passed then we know it will work.